TypeScript.sublime-commands

```diff
@@ -10,5 +10,6 @@
  { "caption" : "TypeScript: Rename", "command": "typescript_rename" },
  { "caption" : "TypeScript: PasteAndFormat", "command": "typescript_paste_and_format" },
  { "caption" : "TypeScript: FormatLine", "command": "typescript_format_line" },
+ { "caption" : "TypeScript: SignatureInfo", "command": "typescript_signature_popup" },
```

Comment:
Maybe put the ``SignatureInfo`` below the ``FormatBlock``? As all the formatting ones were grouped together